editoast/src/views/timetable.rs

```diff
@@ -931,10 +931,10 @@ impl PhysicsConsistParameters {
     pub fn compute_etcs_brake_params(&self) -> Option<EtcsBrakeParams> {
         // TODO: handle towed rolling-stock when applying ERTMS to that case
         assert!(
-            !self.traction_engine.has_etcs_level2() || self.towed_rolling_stock.is_none(),
+            self.traction_engine.get_etcs_brake_params().is_some() || self.towed_rolling_stock.is_none(),
```

Comment:
I'm wondering if we can't just check for towed rolling stock. Because in the case there is not, the function is just fine (and may still return `None`).

```suggestion
            self.towed_rolling_stock.is_none(),
```